src/c++/perf_analyzer/genai-perf/genai_perf/llm_inputs/llm_inputs.py

```diff
@@ -87,7 +93,9 @@ def create_llm_inputs(
         num_of_output_prompts: int = DEFAULT_NUM_PROMPTS,
         add_model_name: bool = False,
         add_stream: bool = False,
-        tokenizer: AutoTokenizer = DEFAULT_TOKENIZER,
+        tokenizer: Union[PreTrainedTokenizer, PreTrainedTokenizerFast] = get_tokenizer(
```

Comment:
nit: we could also use `|` which I think is less verbose 😃

```suggestion
        tokenizer: PreTrainedTokenizer | PreTrainedTokenizerFast = get_tokenizer(
```